### PR TITLE
ci: run start-two-nodes-in-docker.sh test script in CI

### PR DESCRIPTION
**Pull request by SergeTupchiy**
Fixes https://emqx.atlassian.net/browse/EMQX-8884

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` and `.zh.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: .github/workflows/build_slim_packages.yaml [hunk 1]
```diff
@@ -194,6 +194,11 @@ jobs:
         HTTP_PORT=$(docker inspect --format='{{(index (index .NetworkSettings.Ports "18083/tcp") 0).HostPort}}' $CID)
         ./scripts/test/emqx-smoke-test.sh localhost $HTTP_PORT
         docker stop $CID
+    - name: test two nodes cluster with proto_dist=inet_tls in docker
+      run: |
+        ./scripts/test/start-two-nodes-in-docker.sh -P $EMQX_IMAGE_TAG
+        # cleanup
+        ./scripts/test/start-two-nodes-in-docker.sh -c
```
Comment (zmstone) [inline .github/workflows/build_slim_packages.yaml:201]: run smoke test against the haproxy endpoint ?

File: .github/workflows/build_slim_packages.yaml [hunk 2]
```diff
@@ -194,6 +194,11 @@ jobs:
         HTTP_PORT=$(docker inspect --format='{{(index (index .NetworkSettings.Ports "18083/tcp") 0).HostPort}}' $CID)
         ./scripts/test/emqx-smoke-test.sh localhost $HTTP_PORT
         docker stop $CID
+    - name: test two nodes cluster with proto_dist=inet_tls in docker
+      run: |
+        ./scripts/test/start-two-nodes-in-docker.sh -P $EMQX_IMAGE_TAG
```
Comment (zmstone) [inline .github/workflows/build_slim_packages.yaml:199]: We can pick an old image, so to cover the compatibility checks too.
i.e. newer version node joins older version cluster (how rolling upgrade is done).
```suggestion
        ./scripts/test/start-two-nodes-in-docker.sh -P $EMQX_IMAGE_TAG $EMQX_IMAGE_OLD_VERSION_TAG
```

File: .github/workflows/build_slim_packages.yaml (hunk 1 above)
Comment (SergeTupchiy) [reply .github/workflows/build_slim_packages.yaml:205]: Done.

File: .github/workflows/build_slim_packages.yaml (hunk 2 above)
Comment (SergeTupchiy) [reply .github/workflows/build_slim_packages.yaml]: Done.

**Review by zmstone — APPROVED** [review]

**Pull request merged**